examples/polyfill/recipes/default.rb

```diff
@@ -0,0 +1,21 @@
+automat_dir = '/hab/pkgs/chef/automate-ui/2.0.0'
+unless ::File.directory?(automat_dir)
+  raise "Directory #{automat_dir} is missing!"
+end
+files = ::Dir.glob(::File.join(automat_dir, '*'))
+parent_dir = ::File.basename(files.first) # => '20240305055235'
+ 
+ 
+file_names_tobe_deleted = %w(collection chef)
+ 
+ 
+file_names_tobe_deleted.each do |f|
+  file_to_be_deleted = "#{automat_dir}/#{parent_dir}/dist/assets/chef-ui-library/#{f}/sandbox.html"
+  file file_to_be_deleted do
+    action :delete
+  end
+end
+file_to_be_created = '/var/log/chef-quarantine-polyfill'
```

Comment:
am not sure how adding this empty file helps. perhaps as a record of the fact that the recipe has been run?

Reply:
thats right! We'll get to know if the recipe has been executed.

Reply:
chef-client run logs will show the status and running chef-client subsequently will show the resources as up to date so I don't really see the need for this empty file either.